assets/questions/findMostCommonRepeatedCharacter.js

```diff
@@ -0,0 +1,276 @@
+// Copyright 2018 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Most Common Repeated Character.
+ */
+
+globalData.questions['findMostCommonRepeatedCharacter'] = {  // eslint-disable-line dot-notation
+  title: 'Most Common Repeated Character',
+  starterCode: {
+    python:
+`def findMostCommonRepeatedCharacter(s):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`import collections
+
+class AuxiliaryCode(object):
+
+    @classmethod
+    def repeatsOnly(cls, s, ignore_case=True, reset_prev=True):
+      output = []
+      prev = None
+      for c in s:
+        if (prev and ignore_case and c.lower() == prev.lower()) or c == prev:
+          output.append(prev)
+          output.append(c)
+          if reset_prev:
+            prev = None
+        else:
+          prev = c
+      return ''.join(output)
+          
+    @classmethod
+    def overcountMoreThanTwoRepeats(cls, s):
+      repeats = cls.repeatsOnly(s, True, False)
+      if repeats:
+        return collections.Counter(repeats).most_common(1)[0][0]
+          
+    @classmethod
+    def dontIgnorePunctuation(cls, s):
+      repeats = cls.repeatsOnly(s)
+      repeats = [x for x in repeats if (not x.isalpha() and not x.isdigit())]
+      if repeats:
+        return collections.Counter(repeats).most_common(1)[0][0]
+    
+    @classmethod
+    def dontIgnoreCase(cls, s):
+      repeats = cls.repeatsOnly(s, False)
+      if repeats:
+        return collections.Counter(repeats).most_common(1)[0][0]
+    
+    @classmethod
+    def lastRepeatOnly(cls, s):
+      repeats = [x.lower() for x in cls.repeatsOnly(s)]
+      if repeats:
+        return repeats[-1].lower()
+    
+    @classmethod
+    def firstRepeatOnly(cls, s):
+      repeats = [x.lower() for x in cls.repeatsOnly(s)]
+      if repeats:
+        return repeats[0].lower()
+    
+    @classmethod
+    def includeNonRepeated(cls, s):
+      counter = collections.Counter(s)
+      most_common = counter.most_common(1)
+      if most_common:
+        return most_common[0][0]
+`
+  },
+  tasks: [{
+    id: 'findMostCommonRepeatedCharacter',
+    instructions: [{
+      content: [
+        'In this problem, we\'d like you to write a function that determines ',
+        'the most common repeated character in an ASCII string (ignoring ',
```

Comment:
I think there are two questions here.
1. If this is too hard to Fuzzy, we could use the simpler problem but that doesn't require this work because it's already here:
https://github.com/google/tie/blob/master/assets/questions/findMostCommonCharacter.js

2. Whether "aaa" counts as 1 or 2 repeats: I don't care, we can call it out explicitly either way. Counting it as 2 saves one line of code in the solution.

This also leads to a meta question: how does TIE deal with clarifying questions. In the context of interviews, I intentionally leave questions underspecified to test the interviewee's thinking about the problem and clarify later. In this case, we're not dealing with an interview so should I just call out all the edge cases and how they should be handled? It could be an interesting feature in the future to allow the user to specify edge case inputs before they start coding and to have TIE provide the expected answers for those edge cases.